### DISCUSSION TITLE
v86 does weird things when it has too much power

### DISCUSSION THREAD

**Discussion by CorruptionVR** [copy/v86#958] [General]
when i give v86 512 video memory using https://archiveos.org/chromixium/, it completely skipped the boot screen and didn't boot, and when i do that to windows pe 3.0, i get this
![image](https://github.com/copy/v86/assets/78228404/84c03644-9dba-4c95-a990-15c837f6eff0)

**Comment by Alsefina** [comment]
512MB vram seems to break most other guest systems as well. might be due to technical limitations. anything over 2GB of system memory and any OS will also refuse to boot.

**Reply by CorruptionVR** [reply]
ok, so windows PE loaded, and it did this
![image](https://github.com/copy/v86/assets/78228404/82e0217b-1e4a-47ed-8c47-7a8e438ff7bb)
so, note to self. don't do that.
